goldstone/core/models.py

```diff
@@ -2634,3 +2636,70 @@ def integration(cls):
     def resourcetype(cls):
 
         return "routers"
+
+
+class DefinedSearch(models.Model):
+    """Defined searches, both system and user-created."""
+
+    # This object's Goldstone UUID.
+    uuid = UUIDField(version=1, auto=True, primary_key=True)
+
+    name = models.CharField(max_length=64)
```

Comment:
> Does it make sense to differentiate owners of searches?  That could simply be core, compliance, and user for now.  In that case I would make the name,owner tuple unique rather than just the name.

Yes, an owner field needs to be added.  And agreed, it and the name should be a unique_together constraint.